draft-ietf-privacypass-protocol.md

```diff
@@ -208,6 +204,54 @@ Clients that use this directory resource before 1686913811 in UNIX time would us
 second key in the "token-keys" list, whereas Clients that use this directory after
 1686913811 in UNIX time would use the first key in the "token-keys" list.
 
+A complete "token-key" value, encoded as it would be in the Issuer directory,
+would look like the following (line breaks are inserted to fit within the per-line
+character limits):
+
+~~~
+$ echo MIIBUjA9BgkqhkiG9w0BAQowMKANMAsGCWCGSAFlAwQCAqEaMBgGCSqGSIb3DQEBCDAL \
```

Comment:
Is putting echo / openssl commands standard for such document?

Reply:
I don't know if it's standard, but we already do this elsewhere in the document 🤷

Reply:
For the openssl part, yeah. Cool.